src/CRM/Export/KeyedField.php

```diff
@@ -9,8 +9,8 @@ public function __construct($key, $subkey) {
     $this->subkey = $subkey;
   }
 
-  public function value() {
-    if (($values = parent::value(NULL))) {
+  public function value($delta = NULL) {
+    if (($values = parent::value($delta))) {
```

Comment:
The data structure returned by `WrapperField::value()` differs depending on whether you pass `NULL` or something else (ie. the default `0`). So passing anything else than `NULL` to this function probably wouldn’t work.